man/fi_domain.3.md

```diff
@@ -778,6 +779,14 @@ for additional information.
 : The maximum number of authorization keys which can be supported per connectionless
   endpoint.
 
+## Optional Capabilities (optional_caps)
+
+Domain level optional capabilities. The following are the supported optional
+capabilities: None
```

Comment:
See comment on previous patch.  This path basically enables all cap bits as optional.